### PR TITLE
feat: add cli for health status

### PR DESCRIPTION
**Pull request by mahbub570**
This pr provides a status check of various system components, allowing administrators to quickly assess the overall health and operational status of the Harbor registry system.

Command:
`harbor health` : Get the health status of Harbor components

health

### CODE REVIEW

**Comment by bupd** [comment]
@mahbub570 Have a look at #81. which does the same with colors.

**Comment by mahbub570** [comment]
I see, @bupd like this?

 
health

**Comment by bupd** [comment]
Adjust the boudaries of the box

File: pkg/api/health_handler.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	"github.com/goharbor/go-client/pkg/sdk/v2.0/client/health"
+	"github.com/goharbor/harbor-cli/pkg/utils"
+	"github.com/spf13/viper"
+)
+
+func GetHealth() (*health.GetHealthOK, error) {
+	credentialName := viper.GetString("current-credential-name")
+	client := utils.GetClientByCredentialName(credentialName)
+
+	ctx := context.Background()
```
Comment (bupd) [inline pkg/api/health_handler.go:16]:
```suggestion
	ctx, client, err := utils.ContextWithClient()
	if err != nil {
		return err
	}
```

File: pkg/views/health/view.go
Comment (bupd) [inline pkg/views/health/view.go:1]: Modify this to follow `user/list/view.go`.

**Review by bupd — CHANGES_REQUESTED** [review]

**Comment by Althaf66** [comment]
@mahbub570  Make these changes in `pkg/views/health/view.go`

```go
package health

import (
	"fmt"
	"os"

	"github.com/charmbracelet/bubbles/table"
	"github.com/charmbracelet/bubbletea"
	"github.com/goharbor/go-client/pkg/sdk/v2.0/client/health"
	"github.com/goharbor/harbor-cli/pkg/views"
	"github.com/goharbor/harbor-cli/pkg/views/base/tablelist"
)

var columns = []table.Column{
	{Title: "Component", Width: 18},
	{Title: "Status", Width: 26},
}

func PrintHealthStatus(status *health.GetHealthOK) {
	var rows []table.Row
	fmt.Printf("Harbor Health Status:: %s\n", styleStatus(status.Payload.Status))
	for _, component := range status.Payload.Components {
		rows = append(rows, table.Row{
			component.Name,
			styleStatus(component.Status),
		})
	}

	m := tablelist.NewModel(columns, rows, len(rows))

	if _, err := tea.NewProgram(m).Run(); err != nil {
		fmt.Println("Error running program:", err)
		os.Exit(1)
	}
}

func styleStatus(status string) string {
	if status == "healthy" {
		return views.GreenStyle.Render(status)
	}
	return views.RedStyle.Render(status)
}
```

**Comment by Althaf66** [comment]
`pkg/api/health_handler.go`

```go
package api

import (
	"fmt"

	"github.com/goharbor/go-client/pkg/sdk/v2.0/client/health"
	"github.com/goharbor/harbor-cli/pkg/utils"
)

func GetHealth() (*health.GetHealthOK, error) {
	ctx, client, err := utils.ContextWithClient()
	if err != nil {
		return nil, err
	}

	response, err := client.Health.GetHealth(ctx,&health.GetHealthParams{})
	if err != nil {
		return nil, fmt.Errorf("error getting health status: %w", err)
	}

	return response, nil
}

```

**Comment by mahbub570** [comment]
thanks @Althaf66 for your suggestions.

**Comment by Althaf66** [comment]
ready for review

**Review by bupd — APPROVED** [review]
LGTM 🚀

**Pull request merged**